File: MODUL 3/PRAK3/PRAK304-2410817210025-Helga.py
a = int(input());

if a >= 100:
    print("Anda Menginput Melebihi Limit Bilangan");
elif a >= 20:
    print("Puluhan");
elif a >= 11:
    print("Belasan");
elif a >= 1:
    print("Satuan");
else:
    print("Nol");

File: MODUL 3/PRAK 3/PRAK304-2410817210025-Helga.c
#include <stdio.h>
int main(){
    int a;

    scanf("%d", &a);

    if(a >= 100){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    else if(a >= 20){
        printf("Puluhan");
    }
    else if(a >= 11){
        printf("Belasan");
    }
    else if(a >= 1){
        printf("Satuan");
    }
    else{
        printf("Nol");
    }
}